toolkit/tools/pkg/imagecustomizerlib/liveosisobuilder.go

```diff
@@ -315,7 +315,7 @@ func (b *LiveOSIsoBuilder) updateGrubCfg(savedConfigKernelArgsFilePath string, g
 		return fmt.Errorf("failed to update the root kernel argument in the iso grub.cfg:\n%w", err)
 	}
 
-	inputContentString, err = updateSELinuxCommandLineHelperAll(inputContentString, imagecustomizerapi.SELinuxModeDisabled)
+	inputContentString, err = updateSELinuxCommandLineHelperAll(inputContentString, imagecustomizerapi.SELinuxModeDisabled, true /*allowMultiple*/, false /*requireKernelOpts*/)
```

Comment:
nit: Line is a tad long. (I like to wrap at 120.)

Reply:
pending